### DISCUSSION TITLE
Is there a way to get the `slug` from the parent page?

### DISCUSSION THREAD

**Discussion by samtrion** [sveltia/sveltia-cms#149] [Q&A] [answered]
I am currently trying to synchronize documentation directories from different repositories with a few GitHub pipelines. To avoid unnecessary configuration issues, I would like to get the slug from the parent.

Something like `{{parent.slug}}`.

**Comment by kyoshino** [comment]
Hmm, not sure what a parent page means. Any example?

**Reply by kyoshino** [reply]
I was thinking about [nested collections](https://decapcms.org/docs/collection-nested/) but it’s not supported in Sveltia yet. Will work on it in the near future. 🙂 

Or you mean the collection name?

**Reply by samtrion** [reply]
Ya, something like [`nested collections`](https://decapcms.org/docs/collection-nested/).

In the broadest sense, a subpage of a collection.

Collection slug: `foo`
Subpage slug: `{{parent.slug}}bar` => `foobar`

**Reply by kyoshino** [reply, marked as answer]
Okay. I plan to support nested collections in a few months. I guess it’s a bit hard when thinking about i18n support.

For now you can just hard code the collection name in the [`slug`](https://decapcms.org/docs/configuration-options/#slug) option if it’s not deeply nested:

```yaml
collections:
  - name: 'foo'
    label: 'Foo'
    folder: 'src/content/foo'
    slug: 'foo{{slug}}'
```

**Comment by kyoshino** [comment]
FYI:

- [v0.25.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.25.0) added support for [localized entry slugs](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#localizing-entry-slugs) — it depends on your framework
- [v0.30.0](https://github.com/sveltia/sveltia-cms/releases/tag/v0.30.0) added support for German umlauts transliteration in slugs
